jupyterlab_git/git.py

```diff
@@ -326,7 +330,7 @@ def __init__(self, contents_manager):
         note = [0] * 3
         count = 0
         temp = ""
-        line_array = my_output.splitlines()
+        line_array = re.split("\x00|\n|\r\n|\r", my_output)
```

Comment:
why the slightly different split here? Aren't `\r` and `\n` converted to `\x00` by the `-z` flag?

Reply:
Apparently only sometimes. if you try `git log -1 --stat --numstat --oneline -z` you can see that only the first half has the newlines swapped for `\x00`, the second half (from --stat) doesn't seem to be affected by `-z`:
![image](https://user-images.githubusercontent.com/10111092/76376674-41f60680-631f-11ea-8986-095d432035c3.png)

Though the output of --stat only seems to be used to grab the final line of that output, which could be generated from the file parsing that already gets done, so this weird splitting could be eliminated by dropping the `--stat` and changing the parsing logic slightly.